### DISCUSSION TITLE
Is it possible to use Cbc on a Raspberry Pi?

### DISCUSSION THREAD

**Discussion by JochemBruijninckx** [coin-or/Cbc#569] [Discussion]
I have tried following [this documentation](https://python-mip.readthedocs.io/en/latest/install.html) to fetch and build the required binaries, but keep on running into NameError: 'cbclib' is not defined. I am starting to wonder if maybe I am overlooking something and it is simply not possible?

**Comment by Iain-R** [comment]
Hi, It seems implied in the link that it is possible to run on a raspberry pi, and given that it runs linux I cannot see why it wouldn't be possible. 
It seems that others have had a similar error, https://github.com/coin-or/python-mip/issues/167. 
If you are still having errors later in the week then I will try to reproduce them on a pi to help :)

**Reply by tkralphs** [reply]
See 

https://github.com/coin-or/python-mip/issues/276 
https://github.com/coin-or/coinbrew/issues/73 
https://github.com/coin-or/Cbc/issues/501

for more discussion of building on a Raspberry Pi specifically for the purpose of deploying with `python-mip`.
